File: src/main/java/com/tecode/house/liuhao/control/PackageTable.scala
package com.tecode.house.liuhao.control

import com.tecode.table.TablePost
import org.apache.hadoop.hbase.HBaseConfiguration
import org.apache.hadoop.hbase.client.Result
import org.apache.hadoop.hbase.io.ImmutableBytesWritable
import org.apache.hadoop.hbase.mapreduce.TableInputFormat
import org.apache.hadoop.hbase.util.Bytes
import org.apache.spark.{SparkConf, SparkContext}
import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.{RequestMapping, RequestMethod, RequestParam, ResponseBody}

/**
  * Created by Administrator on 2018/12/6.
  */
@Controller
class PackageTable {
  @ResponseBody
  @RequestMapping(value = Array("/test-table"), method = Array(RequestMethod.POST))
  def tabletest(@RequestParam(required = false)tablepost:TablePost,tablename:String)={


  }
}

File: src/main/java/com/tecode/house/liuhao/control/PackageTable.java
package com.tecode.house.liuhao.control;

import com.tecode.house.liuhao.server.getTableServersDao;
import com.tecode.house.liuhao.server.impl.GetTableServersDaoImpl;
import com.tecode.table.Table;
import com.tecode.table.TablePost;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;
import org.springframework.web.bind.annotation.ResponseBody;

/**
 * Created by Administrator on 2018/12/7.
 */
public class PackageTable {
    private getTableServersDao tableserver = new GetTableServersDaoImpl();
    @ResponseBody
    @RequestMapping(value = "/CitySize_Tax_Avg_bar_table", method = RequestMethod.POST)
    public Table getCityTable(TablePost tablePost){
        return tableserver.getCityTax(tablePost);
    }

    @ResponseBody
    @RequestMapping(value = "/basics_structuretype_num_bar_table", method = RequestMethod.POST)
    public Table getStructuceTypeTable(TablePost tablePost){
        return tableserver.getStructureType(tablePost);
    }
}
